os_info/src/redox/mod.rs

```diff
@@ -18,7 +18,7 @@ pub fn current_platform() -> Info {
         os_type: Type::Redox,
         version,
         bitness: Bitness::Unknown,
-        family: Family::DOS,
+        family: Family::WindowsNT,
```

Comment:
This also seems weird to me. 🙃

Reply:
So this one, @kraileth suggested changing since Windows is no longer truly "DOS." He recommended changing it to either something NT since NT is the kernel architecture or just having the family also be Windows.

Reply:
Just a thought here: the `OS` enviroment variable expands to `Windows_NT` on windows systems